spec/helpers/providers_helper_spec.rb

```diff
@@ -19,8 +19,9 @@
     %w[home_addresses
        home_address_lookups
        home_address_selections
-       home_address_different_address_reaons
-       home_address_different_addresses]
+       different_address_reaons
```

Comment:
Is the typo on reasons (`reaons`) deliberate?
I know you didn't add it but, can we fix it on this PR or does it need a separate one?

Reply:
👍 no not a deliberate typo! will fix